sdk/monitor/azure-monitor-query/README.md

```diff
@@ -405,6 +405,9 @@ raw JSON response. For example:
 }
 ```
 
+#### Overcoming Log Analytics query size limitations
+If your query exceeds the service limits, you can refer to the large log query documentation to learn how to overcome them.
```

Comment:
As a rule of thumb, avoid gerunds in headings. Also, this "large log query documentation" text should be a link.
```suggestion
#### Overcome Log Analytics query size limitations

If your query exceeds the service limits, see the large log query documentation to learn how to overcome them.
```

Reply:
@scottaddie I plan on linking via a GitHub permalink once the PR merges.